gha/py-dependencies/action.yaml

```
@@ -55,27 +59,31 @@ runs:
         ./scripts/github/install-virtualenv.sh
         PIP_VERSION=$(grep '^PIP_VERSION' Makefile | awk '{print $3}')
         echo "::set-output name=pip-version::${PIP_VERSION}"
-        ${VIRTUALENV_DIR}/bin/pip install -U "pip==${PIP_VERSION}" setuptools
+        ${{env.VIRTUALENV_DIR}}/bin/pip install -U "pip==${PIP_VERSION}" setuptools
 
     - name: Install StackStorm requirements
       # this should run in the st2 checkout
+      shell: bash
       working-directory: st2
       env:
         VIRTUALENV_DIR: "~/virtualenv"
       run: |
-        ${VIRTUALENV_DIR}/bin/pip install -r requirements.txt
+        ${{env.VIRTUALENV_DIR}}/bin/pip install -r requirements.txt
 
     - name: Install runners
+      shell: bash
       env:
         VIRTUALENV_DIR: "~/virtualenv"
         ACTION_PATH: ${{ github.action_path }}
         ST2_REPO_PATH: st2
+        CI_DIR: ${{ format('{0}{1}', github.action_path, '/../..') }}
       # TODO: This Makefile has CircleCI assumptions
       run: |
-        cp ${ACTION_PATH}/../../.circle/Makefile .
+        cp ${{env.ACTION_PATH}}/../../.circle/Makefile .
```

It turns out, github already exposes action_path as an env var, so we can remove ACTION_PATH from the env block and just use `$GITHUB_ACTION_PATH`
```suggestion
        cp ${GITHUB_ACTION_PATH}/../../.circle/Makefile .
```

---

I just pushed this to the upstream gha branch